backend/fms_core/management/commands/createdocs.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Optional
+from django.core.management.base import BaseCommand, CommandParser
+
+class Command(BaseCommand):
+    help = "Generates documentation for python modules"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--output",
+            help="The directory to output generated documents to (default: ./docs)",
+            type=str,
+            default="./docs"
```

Comment:
You might want to register the default as a path relative to the repo root. This default value may lead to an output being created at undesirable position depending on the location of the execution. Maybe use the settings.py file to set the default at the App level.